frontend/src/store/utils.js

```diff
@@ -2,14 +2,35 @@
 // factories
 export function doRequestFactory(REQUEST, COMMITS) {
   return async function ({ commit }, { action, args, load = 'TOGGLE_LOADING' }) {
```

Comment:
load has a default thought so won't this always resolve?

Reply:
no cos we can pass `''` or `null` which are falsey and won't be overridden by the default

Reply:
actually i'm not 100% sure about `null` but `''` definitely works